Dockerfile.alpine

```diff
@@ -17,8 +17,10 @@ ENV NGINX_PROXY_VERSION=${NGINX_PROXY_VERSION} \
 RUN apk add --no-cache --virtual .run-deps bash openssl
 
 # Configure Nginx
-RUN sed -i 's/worker_connections.*;$/worker_connections   10240;/' /etc/nginx/nginx.conf \
+RUN echo -e "\ninclude /etc/nginx/toplevel.conf.d/*.conf;" >> /etc/nginx/nginx.conf \
```

Comment:
Dockerfile.debian does not contain `-e` on echo. Not sure if we need it here - trailing newline is good thing to have (but maybe I don't know what you do).

Reply:
The `\n` is added here to prevent problems that may occur if the default configuration file of nginx does not contain a newline at the end in future updates.

Reply:
@p12tic `echo` does not behave the same on Debian and Alpine, the latter need the `- e` flag for `\n` to be converted to a newline.

I'm not sure either command result in nginx.conf ending with a newline if that's the desired outcome, the `\n` at the start is rather to insert a blank line between our statement and the end of the existing `http` block.

Reply:
@buchdag In fact, in my tests, Debian's echo still needs `-e` to ensure that `\n` is escaped correctly

```bash
$ docker run --rm -it nginx:1.26.0 bash
root@e0184964fc67:/# echo "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"
\ninclude /etc/nginx/toplevel.conf.d/*.conf;
root@e0184964fc67:/# echo -e "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"

include /etc/nginx/toplevel.conf.d/*.conf;
root@e0184964fc67:/# 
exit

```

```bash
$ docker run --rm -it nginx:1.26.0-alpine sh  
/ # echo "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"
\ninclude /etc/nginx/toplevel.conf.d/*.conf;
/ # echo -e "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"

include /etc/nginx/toplevel.conf.d/*.conf;
/ # 

```

Reply:
@KagurazakaNyaa strangely that's what I initially committed (using the `-e` flag on both version) and it resulted on `/n` being printed to the nginx.conf and [all tests failing for the Debian version](https://github.com/nginx-proxy/nginx-proxy/actions/runs/8969610890).

Reply:
@buchdag I know why this strange problem occurs. During `docker build`, `/bin/sh` is used instead of `/bin/bash`, and their `echo` behaviors are different.

```sh
$ docker run --rm -it nginx:1.26.0 sh
# echo "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"

include /etc/nginx/toplevel.conf.d/*.conf;
# echo -e "\ninclude /etc/nginx/toplevel.conf.d/*.conf;"
-e 
include /etc/nginx/toplevel.conf.d/*.conf;
# 
```

Reply:
So using `echo` without `-e` in debian is correct

Reply:
Is it okay for you if I merge the PR then ?

Reply:
LGTM, if there are no more issues, I think it is ready to merge.